frontend/src/modules/projects/views/projectView.js

```diff
@@ -97,7 +97,10 @@ define(function(require) {
 
     deleteProject: function(event) {
       this.model.destroy({
-        success: _.bind(this.remove, this),
+        success: _.bind(function() {
+          Origin.trigger('dashboard:refresh');
+          this.remove();
+        }, this),
```

Comment:
Please can you switch to native bind?

Reply:
👍 have also updated all other uses of `_.bind` in the file.